project1/index.html

```diff
@@ -6,67 +6,66 @@
-
-    <p>That may include the ability to come up with solutions to the problem of finding food, for example, or a knack
-        for planning for some challenge in the future. Intelligent animals don’t rely on fixed responses to survive —
-        they can invent new behaviors on the fly.</p>
-
-    <p>To measure animal intelligence, scientists observe creatures in the wild — watching a dolphin <a href="https://www.nytimes.com/2011/07/26/science/26obdolphin.html?module=inline">stick
-            a sponge on its beak to avoid getting cuts</a> from sharp rocks and coral, for example. Or they bring
-        animals into the lab and offer them puzzles to solve, such as rewarding crows when they <a href="https://www.nytimes.com/2018/06/28/science/crows-toolmaking.html?module=inline">learn
-            to rip paper into strips of just the right size.</a></p>
-
-    <p>Only a few species stand out in these studies, and by comparing them, scientists have identified some shared
-        factors. The animals have big brains relative to their body size, they live for a long time, and they can form
-        long-lasting social bonds.</p>
+    <p>For decades,
+        researchers have studied how certain animals evolved to be intelligent,
+        among them apes,
+        elephants,
+        dolphins and even some birds,
+        such as crows and parrots.</p>
+    <p>But all the scientific theories fail when it comes to cephalopods,
+        a group that includes octopuses,
+        squid and cuttlefish. Despite feats of creativity,
+        they
+        <a href="https://www.cell.com/trends/ecology-evolution/fulltext/S0169-5347(18)30267-2?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0169534718302672%3Fshowall%3Dtrue">
+            lack some hallmarks</a>of intelligence seen in other species.</p>
+    <p>“It’s an apparent paradox that’s been largely overlooked in the past,
+        ” said Mr. Amodio. He and five other experts on animal intelligence explore this paradox in a paper published
+        this month in the journal Trends in Ecology and Evolution.</p>
+    <p>For scientists who study animal behavior,
+        intelligence is not about acing a calculus test or taking a car apart and putting it back together.
+        Intelligence comprises sophisticated cognitive skills that help an animal thrive.</p>
+    <p>That may include the ability to come up with solutions to the problem of finding food,
+        for example, or a knack for planning for some challenge in the future. Intelligent animals don’t rely on fixed
+        responses to
+        survive — they can invent new behaviors on the fly.</p>
+    <p>To measure animal intelligence,scientists observe creatures in the wild — watching a dolphin <a href="https://www.nytimes.com/2011/07/26/science/26obdolphin.html?module=inline">
+            stick a sponge on its beak to avoid getting cuts</a>from sharp rocks and coral,
```

Comment:
Missing a space between "cuts" and "from":
```suggestion
            stick a sponge on its beak to avoid getting cuts</a> from sharp rocks and coral,
```